nevergrad/benchmark/experiments.py

```diff
@@ -298,22 +298,29 @@ def yabbob(seed: tp.Optional[int] = None, parallel: bool = False, big: bool = Fa
     if not parallel and not small:
         optims += ["Cobyla"]
     optims += ["NGO", "Shiva"]
-    # optims += [x for x, y in ng.optimizers.registry.items() if "chain" in x]
+    if hd and big and not parallel:  # We want to compare the Shiva variants.
+        optims += [x for x, y in ng.optimizers.registry.items() if "hiva" in x]
```

Comment:
are there several optimizers with "hiva" in it? It's really hard to read that way. Depending on the optimizers, there are several options to include them in a readable way